test/messagingsession/DefaultMessagingSession.test.ts

```diff
@@ -130,6 +131,33 @@ describe('DefaultMessagingSession', () => {
       });
     });
 
+    it('Can start with prefetch on', done => {
```

Comment:
Can we add a test where actually the outcome or behavior of prefetch is tested, this is fine to test the connection but is there a change in behavior here?

Reply:
The prefetch feature will send more messages from server side, which is not really a behavior change on client's point of view. However, I have updated the messagingSession demo app in this repo to connect to the server and test this feature e2e.